server/plugin/plugin.go

```diff
@@ -27,6 +28,8 @@ type MatterpollPlugin struct {
 	// setConfiguration for usage.
 	configuration *configuration
 	ServerConfig  *model.Config
+
+	bundle *i18n.Bundle
```

Comment:
I want to reduce places where `i18n.NewLocalizer` func is called, because many localizer with different locale will make me confusing.
So we should have new wrapper i18n.Bundle. 1/5

e.g.:
```
type MatterPollLocalizer struct {
    bundle *i18n.Bundle
    localizer *i18n.Localizer  // Localizer made by DefaultServerLocale
}
func (l *MatterPollLocalizer) localize(...) { ... }
func (l *MatterPollLocalizer) localizeWithUserLocale(u *model.User, ...) { ... }
```

This comment might be related to other comments https://github.com/matterpoll/matterpoll/pull/133/files#r261197956.

Reply:
With the current code, the lifetime of a `i18n.Localizer` is one hook (e.g. `serverHTTP`)

If I understand you correctly, your proposal is to have it the lifetime of the whole plugin instead. Could you elaborate a bit, why do you think this is needed? If I understand the idea of `go-i18n` correctly, `i18n.Localizer` are supposed to be ephemeral.

One additional thought: There are only two places where not the user local is used: When a poll is created and when its deleted. I would argue, that not so often.

To get this straight: I'm not against a Wrapper around `i18n.Localizer`.

Reply:
Thank you for your confirmation.
I did't care about the lifetime of i18n.Localizer and the idea of `go-i18n`. (but I supposed that having the lifetime of whole plugin has no problem.)

I cared about only readability. With one-hook (current approach), sometimes we would check which locale the variable has. 

```
localizer := i18n.NewLocalizer(p.bundle, locale)

...(many sentences)...

localizer.MustLocalizer(message_id, params)  // ...which locale does "localizer" has?
```

With my suggestion, we would be able to know easily which locale will be used.
```
p.locaizer.localize(message_id, params)
p.locaizer.localizeWithUserLocale(user, message_id, params)
```

I don't have strong opinion and I don't intend to reject current approach, because I haven't use `go-i18n` library. 


> One additional thought: There are only two places where not the user local is used: When a poll is created and when its deleted. I would argue, that not so often.

Two places? When searching this branch with `i18n.NewLocalizer`, I got 12 places using it.

Reply:
What if I just rename `localizer` to `localizerUser`? E.g.:

```
localizerUser := i18n.NewLocalizer(p.bundle, locale)

...(many sentences)...

localizerUser.MustLocalizer(message_id, params)  // ...which locale does "localizer" has?
```
Would that help in terms of readability?
 
> Two places? When searching this branch with `i18n.NewLocalizer`, I got 12 places using it.

Sorry, my answer was ambiguous. The point is: A localizer is using the users local every time. But in two places it uses the system language instead.

Reply:
Renaming improve somewhat readability. But if just changing variable name, we'll must remenber the convention of naming when creating new localizer variables. So I like switching locale by method name.
My suggestion would be non-blocking comments, and your code is fine for first commit of localization. So if you don't think that my suggestion is good, please go ahead to merge. Of course, I welcome to continue discussing about this topic.

> Sorry, my answer was ambiguous. The point is: A localizer is using the users local every time. But in two places it uses the system language instead.

It seems that Localizer with server locale is used for `handleVote` and `handleAddOption` too? It will be removed?